### ISSUE TITLE
morebits documentation website is down

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia-gadgets/twinkle#1919]
Website link: https://github.com/wikimedia-gadgets/twinkle/wiki/morebits

Branch: gh-pages - https://github.com/wikimedia-gadgets/twinkle/tree/gh-pages

@Amorymeltzer, I hear you might have set these up originally. Any idea how to fix?

**Comment by bsoyka** [comment]
Fixed!! Pulled the file from https://github.com/wikimedia-gadgets/twinkle.wiki.git using Git. The file was removed on November 1, I'd assume accidentally.

**Comment by NovemLinguae** [comment]
Is it possible someone vandalized the github morebits wiki? https://github.com/wikimedia-gadgets/twinkle/wiki/morebits/_history

Perhaps we should turn off public access to edit it?

Someone else would probably have to do that. I don't think I have access to the settings page.

**Comment by Amorymeltzer** [comment]
@NovemLinguae Yeah, you just found what I was typing out.  No idea at this point—I don't have access anymore, and the gh-pages thang is usually a turn on/off in the settings, so I'd have someone check that to undo whatever was done.  From the [history](https://github.com/wikimedia-gadgets/twinkle/wiki/_history) it looks like the wiki stuff was opened up?  I'd suggest checking the repo settings, but no idea when it would've been set to public in the intervening years.

**Comment by NovemLinguae** [comment]
Hmm, looks like the wiki is fixed, but there's also a github.io site that is broken. Anyone know how to fix that?

https://wikimedia-gadgets.github.io/twinkle/Morebits.html

**Comment by nardog** [comment]
That's #1668.

@siddharthvp [said](https://en.wikipedia.org/w/index.php?title=Wikipedia_talk:Twinkle&oldid=1193060623#c-SD0001-20231230130000-Novem_Linguae-20231230122900):
> This setup currently requires syncing all code changes to the gh-pages and then invoking the docs script. Not an optimal setup – ideally we should automate the whole thing (see #1405).
